atd-vze/src/Components/TableDateRange.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from "react";
+import { withApollo } from "react-apollo";
+import styled from "styled-components";
+import moment from "moment";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { colors } from "../styles/colors";
+
+const StyledDatePicker = styled.div`
+  /* Add Bootstrap styles to picker inputs */
+  .react-datepicker__input-container > input {
+    height: calc(1.5em + 0.75rem + 2px);
+    padding: 0.375rem 0.75rem;
+    font-size: 0.875rem;
+    font-weight: 400;
+    line-height: 1.5;
+    color: ${colors.grey700};
+    background-color: ${colors.white};
+    background-clip: padding-box;
+    border: 1px solid ${colors.grey200};
+    border-radius: 0.25rem;
+  }
+
+  .react-datepicker__day--selecting-range-start {
+    background-color: ${colors.primary} !important;
+  }
+
+  .react-datepicker__day--selecting-range-end {
+    background-color: ${colors.primary} !important;
+  }
+
+  .react-datepicker__day--selected {
+    background-color: ${colors.primary} !important;
+  }
+
+  .react-datepicker__day--in-selecting-range {
+    background-color: ${colors.light};
+    color: ${colors.dark};
+  }
+
+  .react-datepicker__day.react-datepicker__day--in-range {
+    background-color: ${colors.secondary};
+  }
+
+  .react-datepicker__header {
+    background-color: ${colors.light};
+  }
+`;
+
+const TableDateRange = ({ setDateRangeFilter, databaseDateColumnName }) => {
+  const minDate = new Date("2010/01/01"); // TODO add programatic way to insert earliest crash record in DB
```

Comment:
Instead of having the programmatic solution get the oldest record in the DB, perhaps we just provide a useful default date range. Like YTD, getting just the last year of data as the starting place. 

```js
let d = new Date();
d.setFullYear(d.getFullYear() - 1);
```

or with moment.js since you import it already:
```
moment().subtract(1, 'year').format();
```

Reply:
That's a great idea. I like defaulting to a small range of data too. I went with the moment option.